assets/dev/js/frontend/utils/anchor-scroll-margin.js

```diff
@@ -0,0 +1,127 @@
+module.exports = elementorModules.ViewModule.extend( {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				links: '.elementor-element a[href*="#"]',
+				stickyElements: '.elementor-element.elementor-sticky',
+			},
+		};
+	},
+
+	onInit() {
+		const waitForElements = new Promise( ( resolve ) => {
+			// Simulating an async operation
+			setTimeout( () => {
+				resolve();
+			}, 100 );
+		} );
+
+		waitForElements.then( () => {
```

Comment:
Do we need this wait?
If so, then I would prefer to use a method that doesn't include a timeout.